circle.yml

```diff
@@ -182,22 +177,7 @@ workflows:
           tests_group: 2
           tests_type: test.integration
           requires:
-            - python_unit_tests
-            - browser_tests
-            - js_tests
-      - build:
-          name: integration_tests_group_3
-          tests_group: 3
-          tests_type: test.integration
-          requires:
-            - python_unit_tests
-            - browser_tests
-            - js_tests
-      - build:
-          name: integration_tests_group_4
-          tests_group: 4
-          tests_type: test.integration
-          requires:
+            - quality_migrations_tests
```

Comment:
Shouldn't we gate the integration tests with `python_unit_tests`? They will catch any issues before an integration test starts running.

Reply:
Not sure I follow. The integration tests do required python unit tests.

Reply:
Oh yeah, the diff on github was misleading.